Sites/GD2/Project 1/index.html

```diff
@@ -1,71 +1,59 @@
+<!DOCTYPE html>
 <html lang="en">
 
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
-    <link rel="stylesheet" href="Styles.css" </head> <body>
-
+    <link rel="stylesheet" <link rel="stylesheet" href="Styles.css">
```

Comment:
FYI, your styles may be broken because of this line; you need this instead:

`<link rel="stylesheet" href="Styles.css">`